[FEATURE] Heure de fin de session théorique pour le surveillant (PIX-7834)

Steph0 opened the pull request:
## :unicorn: Problème

Afficher coté front l’heure théorique de fin dans un encadré avec l’heure de début.
Heure théorique de fin = heure de début de chaque candidat + durée de la certification qu’il est effectivement en train de passer

## :robot: Proposition

* API `/api/sessions/{id}/supervising`
  * `.included[].attributes.theorical-end-date-time` = `certification-courses.createdAt` + `105min` [`+45min`]

### A débattre

Il n'existe pas de notion de temps associé à une certif, ou au moins de temps supplémentaire pour une complémentaire. Or, il existe des différence par compélmentaire (0 minutes pour CLEA, +45min pour Pix+, et notre PO informe que pour NextGen il n'est pas exclut que il y ai d'autres variantes).
**Devons-nous songer à stocker cela en BDD (dans la table completementary-certifications)**


## :rainbow: Remarques

* Le calcul ne doit bien pas prendre le temps majoré, cette partie là du calcul reste à la charge du surveillant
* Ajout de 45 min pour certif complémentaire, sauf CLEA. Une seule complémentaire possible à ce jour.

## :100: Pour tester

Steph0 committed:
:sparkles: api: Supervisor default session enddate

* [added] Complementary certification key retrieval for additional session time
* [added] Complementary certification key to CertificationCandidateForSupervising
* [added] Theorical end datetime key to CertificationCandidateForSupervising
* [added] Default end datetime computation

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr6128.review.pix.fr)
  * [App (.org)](https://app-pr6128.review.pix.org)
  * [Orga (.fr)](https://orga-pr6128.review.pix.fr)
  * [Orga (.org)](https://orga-pr6128.review.pix.org)
  * [Certif (.fr)](https://certif-pr6128.review.pix.fr)
  * [Certif (.org)](https://certif-pr6128.review.pix.org)
  * [Admin](https://admin-pr6128.review.pix.fr)
  * [API](https://api-pr6128.review.pix.fr/api/)
  * [Pix1D](https://pix1d-pr6128.review.pix.fr)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr6128/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr6128/environment)

P-Jeremy committed:
:pencil2: api: rename complementaryCertification to enrolledComplementaryCertification

P-Jeremy committed:
:pencil2: certif: rename complementaryCertification to enrolledComplementaryCertification

P-Jeremy committed:
:sparkles: api: add userId to CertificationCandidateForSupervising

alexandrecoin committed:
:sparkles: api: check still valid badge acquisition

mariannebost committed:
:sparkles: api: add stillValidBadgeAcquisitions in CertificationCandidateForSupervising model

P-Jeremy committed:
:sparkles: api: add #isStillElligible to CertificationCandidateForSupervising

mariannebost committed:
:sparkles: api: add isStillEligible to sessionForSupervising serializer

alexandrecoin committed:
:sparkles: api: add serialization for userId attribute
This information is necessary to check if the candidate has been reconciliated before the eligibility check

alexandrecoin committed:
:sparkles: certif: add warning message for non longer eligible candidates

alexandrecoin committed:
:globe_with_meridians: certif: translate eligibility warning

Steph0 committed:
:sparkles: api: complementary extratime in DB

* [added] Column in database to introduce complementary certification extratime
* [modified] supervisor theorical end datetime depending on certif extratime

Steph0 committed:
:sparkles: api: complementary certif extratime

Steph0 commented:
Je ferme, réouvert avec un autre rebase fait à part pour repartir suite au merge de 7833 + des modfis de seeds
